src/pages/assets/css/ImageButton.css

```diff
@@ -112,3 +112,42 @@
   font-weight: 500;
   font-family: Arial, Helvetica, sans-serif;
 }
+
+.landing-nav {
+  display: flex;
+  flex-direction: row;
+  background-color: #fff;
+  margin: 10px 70px 10px;
+  position: -webkit-sticky;
+  position: sticky;
+  top: 0;
+  z-index: 100;
```

Comment:
![Screenshot (8)_LI](https://user-images.githubusercontent.com/30633846/130354742-bdafe2ea-5962-48b9-ac99-27523480f2a5.jpg)
I don't know why you used z-index here. z index is definitely not needed here as its suppose to be on the same surface like the rest of the components on that page. Using the z-index has broken the navbar.